pravega-cli/src/main/java/io/pravega/tools/pravegacli/commands/PasswordFileCreator/PasswordFileCreatorCommand.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.pravega.tools.pravegacli.commands.PasswordFileCreator;
```

Comment:
Please, respect the code guidelines: package names are not camel-cased. Also, please move this command to a new package called `admin` or something similar, as this is a fundamentally different command compared to other ones (e.g., Controller, Bookkeeper).